README.md

```diff
@@ -51,16 +87,21 @@ module "lambda" {
 | region                           | AWS Region                         | -                   | yes                                  |
 | component| The component for which the load balancer is being created    |- | yes|
 | deployment_identifier|An identifier for this instantiation                                           |- | yes |
+| deployment_label | A unique label per deployment | - | yes |
+| deployment_type | The type of deployment, e.g. development, production | - | | yes|
 | account_id|AWS account ID                                           |- | yes |
 | vpc_id|VPC to deploy lambda to                                           |- | yes |
+| deploy_in_vpc | A boolean flag to disable VPC deployment | true | no
```

Comment:
Description: "Whether or not to deploy the lambda into a VPC (\"yes\" or \"no\")"
Default: yes